llvm/lib/Target/AMDGPU/AsmParser/AMDGPUAsmParser.cpp

```diff
@@ -3005,7 +3006,7 @@ bool
 AMDGPUAsmParser::isNamedOperandModifier(const AsmToken &Token, const AsmToken &NextToken) const {
   if (Token.is(AsmToken::Identifier) && NextToken.is(AsmToken::LParen)) {
     const auto &str = Token.getString();
-    return str == "abs" || str == "neg" || str == "sext";
```

Comment:
Today I learned that we already accept `abs(...)` and `neg(...)`. I had no idea!

Reply:
Yeah, we do. Yet another compatibility thing. There is a special handling of the neg(abs()), the only useful combination. If we ever want to handle lit(abs()) it would need a similar special case handling.